MaterialDesignExtensions.sln

```diff
@@ -1,56 +1,56 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2026
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29613.14
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MaterialDesignExtensions", "MaterialDesignExtensions\MaterialDesignExtensions.csproj", "{809632DA-5EB8-4EE8-AD71-57239701550C}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MaterialDesignExtensions", "MaterialDesignExtensions\MaterialDesignExtensions.csproj", "{91016CBF-6DCE-4EF8-9734-CE5D79D8DD27}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MaterialDesignExtensionsDemo", "MaterialDesignExtensionsDemo\MaterialDesignExtensionsDemo.csproj", "{279D7E62-1206-4928-890E-00F51CD0181C}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MaterialDesignExtensionsBuildUtility", "MaterialDesignExtensionsBuildUtility\MaterialDesignExtensionsBuildUtility.csproj", "{44C7BF26-8A25-4045-B5E2-9FD8621AEECF}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MaterialDesignExtensionsTests", "MaterialDesignExtensionsTests\MaterialDesignExtensionsTests.csproj", "{DCF16057-4955-48F1-BD0E-28D671DF59B0}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MaterialDesignExtensionsDemo", "MaterialDesignExtensionsDemo\MaterialDesignExtensionsDemo.csproj", "{C289C816-9D92-4E66-938C-3861968C1A78}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MaterialDesignExtensionsBuildUtility", "MaterialDesignExtensionsBuildUtility\MaterialDesignExtensionsBuildUtility.csproj", "{2E40D411-CE99-489E-A1E4-9F28A1399CA1}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MaterialDesignExtensionsTests", "MaterialDesignExtensionsTests\MaterialDesignExtensionsTests.csproj", "{89B46FF7-5C2F-4768-B1F9-6B29D3D531EB}"
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
-		DebugLongPath|Any CPU = DebugLongPath|Any CPU
+		Debug|x64 = Debug|x64
```

Comment:
Why a 64 bit build? The library does not use any native dependencies so _Any CPU_ should be sufficient. Applications using the library may then decide for themselves if they want to be run as 32 or 64 bit application.